app/league/[leagueId]/entry/all/page.tsx

```diff
@@ -2,7 +2,6 @@
 // Licensed under the MIT License.
 
 'use client';
-
```

Comment:
nit: Why was this removed?

Reply:
I'm not sure why it was removed. I can add the space back in?